### ISSUE TITLE
%property (strong) 

### ISSUE THREAD

**Issue by rob311fan** [theos/logos#26]
I'm trying to make a new property for an ARC project using %property (strong) and I get a compile error of "unrecognized type". It seems like only non-arc types work,

**Comment by uroboro** [comment]
Use `retain` instead until we add proper support for it.

**Comment by kirb** [comment]
`strong` is a keyword specific to the compiler so it can generate the right memory management code, the ObjC runtime only supports `retain`, `copy`, and `assign` for associated objects. Till we add an alias for `strong` to `retain` (see #1), you can use `strong` in your `@property` and `retain` in your `%property`, like this:

```logos
@interface MyClass ()
@property (nonatomic, strong) AnotherClass *awesome;
@end

%hook MyClass
%property (nonatomic, retain) AnotherClass *awesome;
%end
```

**Comment by rob311fan** [comment]
I see, Thanks for the info kirb and uroboro

**Comment by uroboro** [comment]
Fixed in c0dcd06.
